python/ql/src/Security/CWE-327/PyOpenSSL.qll

```diff
@@ -80,6 +86,9 @@ class PyOpenSSL extends TlsLibrary {
   override ProtocolRestriction protocol_restriction() { result instanceof SetOptionsCall }
 
   override ProtocolUnrestriction protocol_unrestriction() {
-    result instanceof UnspecificPyOpenSSLContextCreation
+    result instanceof UnspecificPyOpenSslContextCreation
   }
 }
+
+/** DEPRECATED: Alias for PyOpenSsl */
+deprecated class PyOpenSSL = PyOpenSsl;
```

Comment:
```suggestion
```

this class is not importable by anything outside the CWE-327 directory, so doesn't need a deprecation.